OPTIONAL: Additional Movement + Disable default page actions

SolarPH opened the pull request:
Changes made:
- Disabled default page actions, so that previous hotkeys would be ignored, like ArrowUp to scroll up, and ArrowDown to scroll down.
- Added ArrowUp and ArrowDown to control the tetriminoes, may save some player's fingers and wrists, while giving space 😉
- Added a code so that ArrowUp and ArrowDown, as well as future non-letter keys, would not be attempted to be lowercased, preventing possible extension crash

Test first before merge, if possible 👀

SolarPH committed:
Additional Movement + Disable default page actions

Changes made:
- Disabled default page actions, so that previous hotkeys would be ignored, like ArrowUp to scroll up, and ArrowDown to scroll down.
- Added ArrowUp and ArrowDown to control the tetriminoes, may save some player's fingers and wrists, while giving space 😉
- Added a code so that ArrowUp and ArrowDown, as well as future non-letter keys, would not be attempted to be lowercased, preventing possible extension crash

TEST FIRST BEFORE MAIN MERGE 👀

TreacherousDev commented:
This solved pretty much the main issue why I had to use a different convention than WASD or arrows. Will check on this later. Salamat ulit! <3

TreacherousDev committed:
0.2.0

Added event listeners to disable default page action upon activation
Added blinking effect of full columns before clearing

Changed Hotkeys:
Move Up: (W)
Move Down: (S)
Rotate Counterclockwise: (A)
Rotate Clockwise: (D)
Instant Drop: (Space)
